PWGCF/EbyEFluctuations/Tasks/netchargeFluctuations.cxx

```diff
@@ -186,39 +190,40 @@ struct NetchargeFluctuations {
-    histogramRegistry.add("rec_hDcaXY_before", "", kTH1D, {dcaAxis});
-    histogramRegistry.add("rec_hDcaZ_before", "", kTH1D, {dcazAxis});
-    histogramRegistry.add("rec_hTPCchi2perCluster_before", "TPC #Chi^{2}/Cluster", kTH1D, {tpcChiAxis});
-    histogramRegistry.add("rec_hITSchi2perCluster_before", "ITS #Chi^{2}/Cluster", kTH1D, {itsChiAxis});
-    histogramRegistry.add("rec_hTPCCrossedrows_before", "Crossed TPC rows", kTH1D, {crossedRowAxis});
-    histogramRegistry.add("rec_hVtxZ_after", "", kTH1F, {vtxzAxis});
     histogramRegistry.add("gen_hVtxZ_after", "", kTH1F, {vtxzAxis});
-    histogramRegistry.add("rec_hDcaXY_after", "", kTH1D, {dcaAxis});
-    histogramRegistry.add("rec_hDcaZ_after", "", kTH1D, {dcazAxis});
-    histogramRegistry.add("rec_hTPCchi2perCluster_after", "TPC #Chi^{2}/Cluster", kTH1D, {tpcChiAxis});
-    histogramRegistry.add("rec_hITSchi2perCluster_after", "ITS #Chi^{2}/Cluster", kTH1D, {itsChiAxis});
-    histogramRegistry.add("rec_hTPCCrossedrows_after", "Crossed TPC rows", kTH1D, {crossedRowAxis});
     histogramRegistry.add("gen_hEta", "", kTH1F, {etaAxis});
-    histogramRegistry.add("rec_hEta", "", kTH1F, {etaAxis});
     histogramRegistry.add("gen_hSign", "", kTH1F, {signAxis});
     histogramRegistry.add("gen_hPt", "", kTH1F, {ptAxis});
-    histogramRegistry.add("rec_hPt", "", kTH1F, {ptAxis});
-    histogramRegistry.add("rec_hPtDcaXY_after", "hPtDCAxy", kTH2D, {ptAxis, dcaAxis});
-    histogramRegistry.add("rec_hPtDcaZ_after", "hPtDCAz", kTH2D, {ptAxis, dcazAxis});
-    histogramRegistry.add("rec_hCentrality", "", kTH1D, {centAxis});
-    histogramRegistry.add("gen_hCentrality", "", kTH1D, {centAxis});
-    histogramRegistry.add("rec_hMultiplicity", "", kTH1D, {multAxis});
-    histogramRegistry.add("gen_hMultiplicity", "", kTH1D, {multAxis});
-  }
 
+    histogramRegistry.add("nch", "", kTH1D, {nchAxis});
+    histogramRegistry.add("nch0", "", kTH1D, {nchAxis});
+    histogramRegistry.add("nch1", "", kTH1D, {nchAxis});
+    histogramRegistry.add("nch2", "", kTH1D, {nchAxis});
+    histogramRegistry.add("nch3", "", kTH1D, {nchAxis});
+    histogramRegistry.add("nch4", "", kTH1D, {nchAxis});
+    histogramRegistry.add("nch5", "", kTH1D, {nchAxis});
+    histogramRegistry.add("nch6", "", kTH1D, {nchAxis});
+    histogramRegistry.add("nch7", "", kTH1D, {nchAxis});
+    histogramRegistry.add("nch8", "", kTH1D, {nchAxis});
+    histogramRegistry.add("nch9", "", kTH1D, {nchAxis});
+    histogramRegistry.add("nch_pos", "", kTH1D, {nchAxis});
+    histogramRegistry.add("nch_neg", "", kTH1D, {nchAxis});
+    histogramRegistry.add("nch_negpos", "", kTH1D, {nchpAxis});
+
+    histogramRegistry.add("nch_cent", "", kTH2D, {centAxis, nchAxis});
+    histogramRegistry.add("nch_pos_cent", "", kTH2D, {centAxis, nchAxis});
+    histogramRegistry.add("nch_neg_cent", "", kTH2D, {centAxis, nchAxis});
+    histogramRegistry.add("nch_negpos_cent", "", kTH2D, {centAxis, nchpAxis});
+  }
   template <RunType run, typename C>
```

Comment:
I would suggest not using `run` because is misleading. `run` has concrete connotations
I would suggest `kindOfRun` or something similar. All along the different places